src/main/resources/assets/neicustomdiagram/lang/zh_CN.lang

```diff
@@ -126,11 +126,11 @@ neicustomdiagram.generators.gregtech5.materialparts.pipesslot=管道
 neicustomdiagram.generators.gregtech5.materialparts.specialpipesslot=特殊管道
 
 neicustomdiagram.generators.gregtech5.materialtools.groupname=GT材料工具
-neicustomdiagram.generators.gregtech5.materialtools.handlelabel=工具手柄
+neicustomdiagram.generators.gregtech5.materialtools.handlelabel=工具手柄材料：%s
```

Comment:
I think the left side is correct for this one; the usage of this string was changed:
![ToolHandleTooltip](https://user-images.githubusercontent.com/54219287/139574923-c2ea7cb7-5c14-416e-a918-adf517ec1fae.png)

It no longer mentions the handle material